src/app/components/cookie-policy/cookie-policy-component.spec.ts

```diff
@@ -0,0 +1,56 @@
+import {Component} from '@angular/core';
```

Comment:
_nitpick:_
Inconsistency with the spacing inside the braces. The ones below are consistent with lint, this one needs adjusting.

Reply:
OK will adjust, and will look at my webstorm settings